tests/scripts/build-release.sh

```diff
@@ -5,7 +5,7 @@ set -ex
 # FUNCTIONS #
 #############
 
-MAKE='build/run make --debug=v --output-sync'
+MAKE='make --debug=v --output-sync'
```

Comment:
Effectively, just stop using the `cross` container and all the magic that goes into it and the `build/run` script. This still results in amd64 and arm64 images being present after the build with `docker images`.